GiEnJul/ClientApp/src/components/admin/editLandingPage/EditMacro.tsx

```diff
@@ -1,8 +1,102 @@
+import { Typography } from '@material-ui/core';
+import { Button, Container, Grid } from '@material-ui/core';
 import * as React from 'react';
+import { useState } from 'react';
+import EditHow from './EditHow';
 import EditQuestions from './EditQuestions';
+import {  EditorState } from 'draft-js';
+import EditCompany from './EditCompany';
+import EditContact from './EditContact';
+import EditStart from './EditStart';
 
-export default () => (
-        <div>
-            <EditQuestions></EditQuestions>
-        </div>
-);
\ No newline at end of file
+export default () => {
+    const [step, setStep] = useState<Number>(1);
+    
+    const [howState, setHowState] = useState<EditorState>(EditorState.createEmpty());
+    const [howTitleState, setHowTitleState] = useState<string>("");
+
+    const [questionState, setQuestionState] = useState<EditorState>(EditorState.createEmpty());
+    const [questionTitleState, setQuestionTitleState] = useState<string>("");
+
+    const [companyState, setCompanyState] = useState<EditorState>(EditorState.createEmpty());
+    const [companyTitleState, setCompanyTitleState] = useState<string>("");
+
+    const [contactState, setContactState] = useState<EditorState>(EditorState.createEmpty());
+    const [contactTitleState, setContactTitleState] = useState<string>("");
+
+    const [startState, setStartState] = useState<EditorState>(EditorState.createEmpty());
+    const [startTitleState, setStartTitleState] = useState<string>("");
+    //these states will be implemented later
+    // const [saving, setSaving] = useState<boolean>(false);
+
+    
+    return (
+        <Container>
+            <Typography align="center" variant='h4' >Rediger gienjul.no</Typography>
+            <Grid
+                container
+                direction="row"
+                justify="center"
+                alignItems="center">
+                <Grid item xs={2}>
```

Comment:
These nav buttons could've been placed inside a for loop, where the button names is looped through. The step variable could have been set by using the loop index.

Reply:
changes implemented